doc/source/limits.rst

```diff
@@ -1,12 +1,159 @@
-=================
-Known limitations
-=================
+=====================================
+Known limitations and important notes
+=====================================
 
 
-As of July 2019
-===============
+As of September 2020
+====================
 
 * No TensorFlow integration
 * Currently only supports ImageNet
-* Dataset storage is not yet normalized as a single file
-* Unknown effect of compression algorithm on model accuracy
+* Unknown effect on model accuracy of transcoding from various JPEG formats to
+  H.265
+* Current transcoding filters failed on 81 images from which 73 come from the
+  train, split, 1 from the validation split and 7 from the test split. The list
+  of the missing images can be found in `ImagetNet missing images`_.
+* Current transcoding filters required 111 images to first be transcoded to PNG
+  prior to the final H.265 format. 102 images come from the train split, 6 from
+  the validation split and 3 from the test split. The list of the dual
+  transcoding images can be found in `ImagetNet extra transcoding images`_
+* High resolution images stored in the
+  :ref:`bzna_input track of the input samples <imagenet_2012>` are currently
+  not available through the :class:`Dataloader`. Their varying size prevent
+  them from being decoded using a single hardware decoder configuration. The
+  selected solution is to represent the images in the HEIF format which will be
+  completed in future development.
+* It is currently not possible to *compose* transformations like you can with
+  ``torchvision.transforms.Compose`` but
+  :py:class:`~benzina.torch.operations.SimilarityTransform` should cover most
+  of the necessary images transformations.
+* :py:class:`~benzina.torch.operations.SimilarityTransform` and
+  :py:class:`~benzina.torch.operations.RandomResizedCrop` slightly differ from
+  the behaviour of ``torchvision.transforms.RandomResizedCrop`` where, instead
+  of falling back to a center crop when the random crop area doesn't fit after
+  10 tries, :class:`SimilarityTransform` will still perform the crop and only
+  center it on the dimension not fitting. Due to the encoding methods used in
+  Benzina, this will usually result in an image with a black top border and a
+  smeared bottom border or a black left border and a smeared right border if
+  the crop area did not fit vertically or horizontally respectively.
+
+ImagetNet missing images
```

Comment:
Agreed I've moved the list of files to the README of the dataset but I've kept some basic and important notes in the doc since ImageNet is currently the only dataset supported.